download-hex.js

```diff
@@ -0,0 +1,36 @@
+(function () {
```

Comment:
No need for self-executing function in node.js. Every module is already wrapped in a function like this.